TrackingBackend/app/tracker.py

```diff
@@ -1,36 +1,44 @@
-from __future__ import annotations
-from .logger import get_logger
-from .config import TrackerConfig
-from app.processes import EyeProcessor, Camera
-from multiprocessing.managers import SyncManager
-from app.utils import clear_queue
 from queue import Queue
 from .types import EyeData
-import cv2
+from cv2.typing import MatLike
+from .logger import get_logger
+from app.utils import clear_queue
+from .config import EyeTrackConfig
+from multiprocessing.managers import SyncManager
+from app.processes import EyeProcessor, Camera, VRChatOSC
 
 logger = get_logger()
 
+
 # TODO: when we start to integrate babble this should become a common interface that eye trackers and mouth trackers inherit from
 class Tracker:
-    def __init__(self, config: TrackerConfig, osc_queue: Queue[EyeData], manager: SyncManager):
+    def __init__(self, config: EyeTrackConfig, uuid: str, manager: SyncManager):
+        self.uuid = uuid
         self.config = config
-        self.uuid = config.uuid
+        self.track_config = config.get_tracker_by_uuid(uuid)
```

Comment:
typo

also, is it critical that we keep them in a list? Maybe it'd be better to keep them in a hashmap since they have a unique UUID already? We'd "save" on time to search and dev UX with prolly pretty similar memory cost

Reply:
The trackers are stored as a array mostly because it was easier to do, i also dont think pydantic is able to save a hashmap, if we do store it as a hashmap we may need to create it at runtime?

Reply:
Possibly, but that was more of a thinking aloud thing rather than let's change that right now, we can back to this later